docs/CodeTransformation.md

```diff
@@ -21,107 +21,107 @@ If you override the `transform` configuration option `babel-jest` will no longer
 
 You can write your own transformer. The API of a transformer is as follows:
 
+<!-- abbreviated: see packages/jest-transform/src/types.ts in https://github.com/facebook/jest for all types -->
```

Comment:
This is just copied from `types.ts`. Unsure if I should add the entire file or as I did here; the main excerpts.